source/common/runtime/runtime_features.cc

```diff
@@ -30,6 +30,7 @@
 // ASAP by filing a bug on github. Overriding non-buggy code is strongly discouraged to avoid the
 // problem of the bugs being found after the old code path has been removed.
 RUNTIME_GUARD(envoy_reloadable_features_allow_alt_svc_for_ips);
+RUNTIME_GUARD(envoy_reloadable_features_allow_metadata_false_disables_metadata);
```

Comment:
nit: this name is little bit hard to read, you could simplify it to `envoy_reloadable_features_disable_metadata`

Reply:
`envoy_reloadable_features_http2_disable_metadata`.